src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -787,6 +795,15 @@ internal async Task InvokeResponse(InvocationResponse invokeResponse)
             if (_executingInvocations.TryRemove(invokeResponse.InvocationId, out ScriptInvocationContext context)
                 && invokeResponse.Result.IsInvocationSuccess(context.ResultSource, capabilityEnabled))
             {
+                var currentInvocationLatency = _workerInvocationStopWatch.GetElapsedTime();
+                _workerChannelLogger.LogDebug($"[HostMonitor] Worker {Id} invocation request took {currentInvocationLatency.TotalMilliseconds}ms");
```

Comment:
Why this new log? Doesn't seem related to the goal of these changes. We have to be careful adding per invocation logs like this because the have a big impact on performance as we've found over time.

Reply:
Removed unnecessary log line.